closure/goog/structs/heap_test.js

```diff
@@ -205,11 +205,39 @@ function testInsertPeek2() {
   var h = new goog.structs.Heap();
 
   h.insert(1, 'b');
-  assertEquals('peak, Should be "b"', h.peek(), 'b');
+  assertEquals('peek, Should be "b"', 'b', h.peek());
   h.insert(3, 'd');
-  assertEquals('peak, Should be "b"', h.peek(), 'b');
+  assertEquals('peek, Should be "b"', 'b', h.peek());
   h.insert(0, 'a');
-  assertEquals('peak, Should be "a"', h.peek(), 'a');
+  assertEquals('peek, Should be "a"', 'a', h.peek());
   h.insert(2, 'c');
-  assertEquals('peak, Should be "a"', h.peek(), 'a');
+  assertEquals('peek, Should be "a"', 'a', h.peek());
+}
+
+function testInsertAllPeek1() {
+  var h1 = new goog.structs.Heap();
+  var h2 = getHeap();
```

Comment:
Absolutely! I wasn't fond of it, either, but figured I'd stay consistent with the rest of the cases. I'll change the tests up as you mentioned to make them more readable.